### PR TITLE
Add schema registry and pandaproxy client node configuration

### PR DESCRIPTION
**Pull request by RafalKorepta**
Console shows the following error
```
{"level":"fatal","ts":"2023-09-27T15:55:53.821Z","msg":"failed to create kafka service","error":"failed to verify connectivity to schema registry: Get \"https://redpanda-0.redpanda.redpanda.svc.cluster.local.:8081/subjects\": context deadline exceeded (Client.Timeout exceeded while awaiting headers)"}
```

That error comes form the missing Redpanda miss configuration.

Fixes https://github.com/redpanda-data/helm-charts/issues/752

### CODE REVIEW

**Review by alejandroEsc — APPROVED** [review]
Nothing blocking, lgtm, just two questions.

File: charts/redpanda/values.schema.json [hunk 1]
```diff
@@ -1405,6 +1405,76 @@
         },
         "rpk": {
           "type": "object"
+        },
+        "schema_registry_client": {
```
[the remark this answers is not in the document]
Comment (RafalKorepta) [reply charts/redpanda/values.schema.json:1409]: Yea, they came strait from client https://docs.redpanda.com/current/reference/node-configuration-sample/

as per 
https://github.com/redpanda-data/helm-charts/blob/404184239489bbbe057e19fa37f4dbac412960e8/charts/redpanda/values.yaml#L1042
and
https://github.com/redpanda-data/helm-charts/blob/404184239489bbbe057e19fa37f4dbac412960e8/charts/redpanda/values.yaml#L1000

File: charts/redpanda/templates/_example-commands.tpl [hunk 2]
```diff
@@ -51,9 +51,9 @@ and tested in a test.
 {{- $flags := fromJson (include "rpk-flags" .) -}}
 {{- $dummySasl := .dummySasl -}}
 {{- if $dummySasl -}}
-{{ .rpk }} topic create test-topic {{ include "rpk-flags-no-admin-no-sasl" . }} {{ include "rpk-dummy-sasl" . }}
+{{ .rpk }} topic create test-topic -p 3 -r {{ .Values.statefulset.replicas | int64 }} {{ include "rpk-flags-no-admin-no-sasl" . }} {{ include "rpk-dummy-sasl" . }}
```
Comment (joejulian) [inline charts/redpanda/templates/_example-commands.tpl:54]: Why didn't this fail on our one-node clusters?

Comment (joejulian) [reply charts/redpanda/templates/_example-commands.tpl:54]: never mind

**Pull request merged**